lib/Vimscript/FFI.idr

```diff
@@ -31,6 +50,9 @@ data VIM_Foreign
   | VIM_ListConcat
   | VIM_ListSetAt
   | VIM_BuiltIn String
+  | VIM_Get VIM_MutableRef String
+  | VIM_Set VIM_MutableRef String
+  | VIM_Toggle VIM_MutableRef String
```

Comment:
Here `VIM_Toggle` takes a `VIM_MutableRef`, which means it can be a register as well. Can registers be toggled?

Reply:
No, only options can -- good catch! -- but I don't think all options can either. Nevertheless, restricting it to options alone using the types sounds good to me.